src/grt/config/win32.c

```diff
@@ -200,8 +200,12 @@ grt_save_backtrace (struct backtrace_addrs *bt, int skip)
 {
   CONTEXT ctxt;
 
+#ifndef NO_FULL_WINAPI_SUPPORT
   RtlCaptureContext (&ctxt);
   get_bt_from_context (bt, &ctxt, skip + 1);
+#else
+  fprintf (stderr, "RtlCaptureContext currently not supported");
+#endif
```

Comment:
Being curious: which platform does not support RtlCaptureContext ?

Reply:
This was for i586-mingw32, which lacks some API features. Maybe it would be better to revert that and not support i586-mingw32 at all (there are too many workarounds involved, really). The Win32 cross build currently bases on i686-mingw-w64-mingw32 which has the better (and maintained) Win32 API support.

Reply:
Yes, please revert it if i586-mingw32 is not really supported.